OIPA/api/city/views.py

```diff
@@ -5,11 +5,40 @@
 
 
 class CityList(ListAPIView):
+    """
+    Returns a list of IATI Cities stored in OIPA.
+
+    ## Result details
+
+    Each result item contains short information about city including URI
+    to city details.
+
```

Comment:
request parameters: 
`fields` request parameter is available here.
aggregation is possible. `/api/cities?fields[aggregations]=total_budget`